core/src/main/java/psiprobe/beans/ContainerWrapperBean.java

```diff
@@ -127,7 +127,7 @@ public void setWrapper(Wrapper wrapper) {
       }
     } catch (ThreadDeath e) {
```

Comment:
`catch (ThreadDeath)` is only necessary if you're going to use `catch (Throwable)`. ThreadDeath extends Error, so it won't be caught by `catch (Exception)`. You can remove `catch (ThreadDeath)` entirely, and it'll be cleaner.

There are many instances of this throughout this PR. I won't note them all because that's just noisy.